metapool/count.py

```diff
@@ -231,10 +236,10 @@ def run_counts(run_dir, metadata):
             minimap2_counts(run_dir, metadata)])
 
     # convenience columns to assess sample quality
-    out['fraction_passing_quality_filter'] = (out['quality_filtered_reads'] /
-                                              out['raw_reads'])
+    good_ratio = out['quality_filtered_reads_r1r2'] / out['raw_reads_r1r2']
```

Comment:
why is this good and where is the bad ratio?

Reply:
Point taken. :) renamed to 'ratio'.